src/textual/_animator.py

```diff
@@ -48,6 +60,7 @@ class SimpleAnimation(Animation):
     end_value: float | Animatable
     final_value: object
     easing: EasingFunction
+    on_complete: Callable[[], None] | None = None
```

Comment:
This may need to be typed as a "CallbackType"

Reply:
Updated and extracted CallbackType to `textual._types` since it's now used in more than one module.